### PR TITLE
Up-level readme changes

### PR DESCRIPTION
**Pull request by dubeyShivank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,61 @@
-# Dolby.io Events React App
+# Dolby.io Live Events React App
+
+## Overview
+
+This project dem
+onstrates what an events experience is like, built using React.
```
Comment (tommydolby) [inline README.md:6]:
```suggestion
This project demonstrates what an events experience is like, built using React.
```

File: README.md [hunk 2]
```diff
@@ -1,23 +1,61 @@
...
 
-You can experience the Dolby.io events app without needing to clone and build the app using the `Deploy to Netlify` button. You will need:
+You can deploy the Dolby.io Virtual Events app without needing to clone and build the app using the Deploy to Netlify button. You will need:
 
-- A Netlify account to which you're logged into
-- Your Dolby.io App key and secret
-- [Optional] Publisher and Subcriber tokens for PubNub. This is only needed if you would like to experience chat and promoting viewers to host.
+* A Netlify account to which you're logged into
+* Your Dolby.io App key and secret
+* [Optional] Publisher and Subcriber tokens for PubNub. This is only needed if you would like to experience chat and promoting viewers to host.
```
Comment (tommydolby) [inline README.md:41]: I think they also need the secret key too?

File: README.md [hunk 3]
```diff
@@ -1,23 +1,60 @@
...
+<p align="center">
+    <img src="./events/documentation/banner.jpg" />
+</p>
+
+| Intended use   | Features                                | Tech stack            |
+| -------------- | --------------------------------------- | --------------------- |
+| Live Events    | create, start, stream and stop an event | Typescript/Javascript |
+| Webinars       | A/V controls for hosts and viewers      | HTML/CSS              |
+| Virtual events | Mute camera and microphone              | React                 |
+|                | Screen share                            |                       |
```
Comment (vishalkharge) [inline README.md:16]: We can also include multi-screen share capability.

File: README.md [hunk 4]
```diff
@@ -1,23 +1,60 @@
...
+> * Requires a PubNub account to work on these features.
+
+## Requirements and supported platforms
+
+The app currently supports
+
+* Chrome 100+
+* Edge 100+
+* Safari 111+
+* Firefox 16.3+
```
Comment (vishalkharge) [inline README.md:32]: Safari and Firefox versions are incorrect

**Pull request merged**